scripts/build_kb.sh

```diff
@@ -2,14 +2,21 @@
 REPO_PATH_FILE="repo.path"
 PREPARED_KB="prepared_kb"
 
+ERRORS_FILE="${PWD}/prepare.log"
+touch "$ERRORS_FILE"
+
 export LD_LIBRARY_PATH=./sc-machine/bin
 if [ ! -d "../kb.bin" ]; then
     mkdir ../kb.bin
 fi
 
-python3 kb_scripts/prepare_kb.py "${PWD%/[^/]*}" $PREPARED_KB $REPO_PATH_FILE
+python3 kb_scripts/prepare_kb.py "${PWD%/[^/]*}" $PREPARED_KB $REPO_PATH_FILE "$ERRORS_FILE"
 
-cd ..
-sc-machine/bin/sc-builder -f -c -i $PREPARED_KB/$REPO_PATH_FILE -o kb.bin -s config/sc-web.ini -e sc-machine/bin/extensions
 
+if [[ -f ${ERRORS_FILE} && ! ( -s ${ERRORS_FILE} )]];
```

Comment:
Add output about build steps and failing

Reply:
It is released in gwf2scs translator